seeders/DatabaseSeeder.ts

```diff
@@ -37,12 +37,21 @@ import {
 } from '@/config';
 import { redactProjectKeyValue } from '@/administration/helpers';
 import { Agent } from '@/runs/execution/constants';
+import { ORM } from '@/database';
 
 const USER_EXTERNAL_ID = 'test';
 const PROJECT_API_KEY = `${API_KEY_PREFIX}testkey`;
 
 export class DatabaseSeeder extends Seeder {
   async run(em: EntityManager): Promise<void> {
+    try {
+      return this._run(em);
+    } finally {
+      await ORM.close();
```

Comment:
Since this just hides the actual problem, please add a comment on why we have it.